src/modules/DisplayModule.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "../SynthModule.h"
+
+class DisplayModule: public SynthModule
+{
+	DisplayModule(ModularSynth& synth);
+
+	char mInputValue[50];
+	int mType;
+
+public:
+//	static const ModuleClass moduleClass = ModuleClass::{{moduleClass}};
```

Comment:
Some commented code here. I think I will look into some linter solutions which would make it easier for contributors to match the code style!